modules/admin_manual/pages/configuration/user/oidc/oidc.adoc

```diff
@@ -48,13 +48,49 @@ Setting up ownCloud Server to work with OpenID Connect requires a couple of comp
 +
 * `'http.cookie.samesite' \=> 'None',`
 +
-See `config.sample.php` and {schemeful-samesite-url}[Schemeful Same-Site] for examples and details.
+See xref:configuration/server/config_sample_php_parameters.adoc#define-how-to-relax-same-site-cookie-settings[config.sample.php] and {schemeful-samesite-url}[Schemeful Same-Site] for examples and details.
 +
 * Settings for the OpenID Connect App
 +
-See `config.apps.sample.php` for examples and details.
+See xref:configuration/server/config_apps_sample_php_parameters.html#app-openid-connect-oidc[config.apps.sample.php] for examples and details or see section xref:settings-in-database[Settings in database] below when running clustered setups.
 - Service discovery for the xref:owncloud-desktop-and-mobile-clients[ownCloud Clients]
 
+=== Settings in Database
+
+In case you run a clustered setup, the following method is preferred because it is stateless. The OpenID Connect app checks for settings in the database first. If none is found, it falls back to the settings stored in `config.php`. If a malformed JSON string is found, an error is logged. You have to store your settings as JSON formatted string in the ownCloud database table `oc_appconfig` with following keys: 
```

Comment:
```suggestion
If you run a clustered setup, the following method is preferred because it is stateless. The OpenID Connect app checks for settings in the database first. If none is found, it falls back to the settings stored in `config.php`. If a malformed JSON string is found, an error is logged. You have to store your settings as a JSON formatted string in the ownCloud database table `oc_appconfig` with the following keys: 
```